app/lib/server/methods/sendMessage.js

```diff
@@ -22,20 +21,7 @@ export function executeSendMessage(uid, message) {
 		});
 	}
 
-	if (message.ts) {
```

Comment:
Same here. This code is there in upstream

Reply:
For an offline message, we need to allow the message, even if the `ts` is greater than 60000.

Reply:
Same here. Add same comment in the code 😄